src/renderer/components/cluster-manager/cluster-actions.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import uniqueId from "lodash/uniqueId";
+import { clusterSettingsURL } from "../+cluster-settings";
+import { landingURL } from "../+landing-page";
+
+import { clusterStore } from "../../../common/cluster-store";
+import { broadcastMessage, requestMain } from "../../../common/ipc";
+import { clusterDisconnectHandler } from "../../../common/cluster-ipc";
+import { ConfirmDialog } from "../confirm-dialog";
+import { Cluster } from "../../../main/cluster";
+import { Tooltip } from "../../components//tooltip";
+
+const navigate = (route: string) =>
+  broadcastMessage("renderer:navigate", route);
```

Comment:
IPC-event name could be re-used from here after merge https://github.com/lensapp/lens/pull/2363/files#diff-1e3a2e0b62c1d7d8887b79611121f74db6b5d18f7de8b9627dffa107a939a9caR10

Reply:
Good catch, may need to do some follow up tweeks before rc.2